src/component/loader/DropZone.jsx

```diff
@@ -222,11 +218,24 @@ function DropZone(props) {
               },
             );
             break;
+          case FILES_TYPES.NMREDATA:
+            loadFiles(selectedFilesByExtensions, { asBuffer: true }).then(
+              async (files) => {
+                for (const zipFile of files) {
+                  const unzipResult = await Zip.loadAsync(zipFile.binary);
```

Comment:
I would rather unzip the data in nmredataToNmrium function. The event is LOAD_NMREDATA_FILE so it is more logical to send directly the file which is in the case of a NMReData a ZIP.